lib/galaxy/datatypes/converters/interval_to_pairix_converter.xml

```diff
@@ -0,0 +1,17 @@
+<tool id="CONVERTER_interval_to_pairix" name="Convert Interval to Pairix" version="1.0.0">
+    <!-- <description>__NOT_USED_CURRENTLY_FOR_CONVERTERS__</description> -->
```

Comment:
Can you add the dependencies on bgzip (or is that samtools/htslib ?) and coreutils here ?